src/translations/strings.yaml

```diff
@@ -568,7 +568,7 @@ onboarding:
     value: "Onboarding step “%1” (%2 of %3). You are currently on this step."
     comment: Accessibility read out for the current step in the step progress bar. %1 is the name of the step. %2 is the index of the current step. %3 is the total number of steps.
   progressBarAccessibilityStepIncomplete:
```

Comment:
Modifying an already-translated string requires a new string id. I usually just tack on "2" at the end